index.js

```diff
@@ -11,6 +11,21 @@ const changePrototype = (to, from) => {
 	Object.setPrototypeOf(to, fromPrototype);
 };
 
+const wrappedToString = (fromBody, name) => {
+	const withToName = name === '' ? '' : `with ${name}() `;
+	return `/* Wrapped ${withToName}*/\n${fromBody}`;
+};
+
+const changeToString = (to, from, name) => {
+	const value = wrappedToString.bind(null, from.toString(), name.trim());
+	Object.defineProperty(to, 'toString', {
+		value,
+		enumerable: false,
```

Comment:
We need to ensure `toString` is not enumerable, otherwise it prints weird in Node.js.